packages/components/tooltip/src/tooltip.vue

```diff
@@ -207,6 +208,12 @@ export default defineComponent({
       return popperContent && popperContent.contains(document.activeElement)
     }
 
+    onDeactivated(() => {
+      if (open.value) {
```

Comment:
Maybe it should not be a simple judgment to open then close it.

Should determine whether the DOM element that triggered the popup still exists.

Reply:
Why do we need to know if dom exists?
If the onDeactivated function is executed, the component is to unAcvived.
Sorry, it's my first time to pr in element.
Thank you very much for answering my questions.

Reply:
It seems that you are right, I was overthinking

Reply:
Thanks for your help anyway